config.mk

```diff
@@ -242,7 +242,12 @@ ifeq ($(WITH_TLS),yes)
 	BROKER_LDADD:=$(BROKER_LDADD) -lssl -lcrypto
 	CLIENT_CPPFLAGS:=$(CLIENT_CPPFLAGS) -DWITH_TLS
 	LIB_CPPFLAGS:=$(LIB_CPPFLAGS) -DWITH_TLS
-	LIB_LIBADD:=$(LIB_LIBADD) -lssl -lcrypto
+	ifeq ($(QNX_TARGET), '' )
```

Comment:
This doesn't work for me on Linux. If I use `ifeq ($(QNX_TARGET),)` then it is ok. Does that work for you as well?